pkg/storage/bloom/v1/tokenizer.go

```diff
@@ -97,17 +97,17 @@ type PrefixedTokenIter struct {
 	buf       []byte
 	prefixLen int
 
-	NGramTokenIter
+	Iterator[[]byte]
```

Comment:
I wouldn't make this field exported.

Reply:
Done

Reply:
it's indirectly  exported. We need have that way so we don't need to reimplement Next, Err...

Reply:
I don't have a stong opinion on this, i'm fine with making this private and implementing Err and Next as wrappers. But I think this is more idiomatic